Amazon.IonDotnet/Internals/Binary/SymbolTableReader.cs

```diff
@@ -527,64 +527,64 @@ public void Dispose()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static int StateFollowingLocalSymbols() => S_STRUCT_CLOSE;
+        private static int StateFollowingLocalSymbols() => StructClose;
 
         private static string GetStateName(int state)
         {
             switch (state)
             {
                 default: return $"UnrecognizedState:{state}";
-                case S_BOF: return "S_BOF";
-                case S_STRUCT: return "S_STRUCT";
-                case S_IN_STRUCT: return "S_IN_STRUCT";
-                case S_NAME: return "S_NAME";
-                case S_VERSION: return "S_VERSION";
-                case S_MAX_ID: return "S_MAX_ID";
-                case S_IMPORT_LIST: return "S_IMPORT_LIST";
-                case S_IN_IMPORTS: return "S_IN_IMPORTS";
-                case S_IMPORT_STRUCT: return "S_IMPORT_STRUCT";
-                case S_IN_IMPORT_STRUCT: return "S_IN_IMPORT_STRUCT";
-                case S_IMPORT_NAME: return "S_IMPORT_NAME";
-                case S_IMPORT_VERSION: return "S_IMPORT_VERSION";
-                case S_IMPORT_MAX_ID: return "S_IMPORT_MAX_ID";
-                case S_IMPORT_STRUCT_CLOSE: return "S_IMPORT_STRUCT_CLOSE";
-                case S_IMPORT_LIST_CLOSE: return "S_IMPORT_LIST_CLOSE";
-                case S_AFTER_IMPORT_LIST: return "S_AFTER_IMPORT_LIST";
-                case S_SYMBOL_LIST: return "S_SYMBOL_LIST";
-                case S_IN_SYMBOLS: return "S_IN_SYMBOLS";
-                case S_SYMBOL: return "S_SYMBOL";
-                case S_SYMBOL_LIST_CLOSE: return "S_SYMBOL_LIST_CLOSE";
-                case S_STRUCT_CLOSE: return "S_STRUCT_CLOSE";
-                case S_EOF: return "S_EOF";
+                case Bof: return "S_BOF";
```

Comment:
These name strings should be changed accordingly.

Reply:
Good catch!

Done.